filer/admin/patched/admin_utils.py

```diff
@@ -106,12 +98,12 @@ def collect(self, objs, source=None, source_attr=None, **kwargs):
                 self.add_edge(None, obj)
             self.model_objs[obj._meta.model].add(obj)
         try:
-            return super(NestedObjects, self).collect(objs, source_attr=source_attr, **kwargs)
+            return super().collect(objs, source_attr=source_attr, **kwargs)
         except models.ProtectedError as e:
             self.protected.update(e.protected_objects)
 
     def related_objects(self, related, objs):
-        qs = super(NestedObjects, self).related_objects(related, objs)
+        qs = super().related_objects(related, objs)
```

Comment:
This https://github.com/divio/django-filer/pull/1202/files#diff-71f149c65e62bb3b8fdcd50b4b00a845R106 is needed here